imagedephi/main.py

```diff
@@ -41,9 +41,7 @@ def imagedephi(ctx: click.Context, override_rules: TextIO | None) -> None:
 
 
 @imagedephi.command
-@click.argument(
-    "input-dir", type=click.Path(exists=True, file_okay=False, readable=True, path_type=Path)
-)
+@click.argument("input_path", type=click.Path(exists=True, readable=True, path_type=Path))
```

Comment:
```suggestion
@click.argument("input-path", type=click.Path(exists=True, readable=True, path_type=Path))
```

To be consistent with `"output-dir"` and with how CLIs typically name things.